File: src/main/java/com/elf/vipForElf/domain/test/vo/AreaListVO.java
package com.elf.vipForElf.domain.test.vo;

import com.elf.vipForElf.domain.test.entity.AreaEntity;
import lombok.Value;

@Value
public class AreaListVO {
    Long id;
    String officeName;
    String roomNumber;

    public AreaListVO(AreaEntity areaEntity){
        this.id= areaEntity.getId();
        this.officeName = areaEntity.getOfficeName();
        this.roomNumber = areaEntity.getRoomNumber();
    }
}

File: src/main/java/com/elf/vipForElf/domain/area/vo/AreaListVO.kt
package com.elf.vipForElf.domain.area.vo

import com.elf.vipForElf.domain.area.entity.AreaEntity

data class AreaListVO(
    val id: Long,
    val officeName: String,
    val roomNumber: String
) {
    constructor(areaEntity: AreaEntity) : this(
        id = areaEntity.id,
        officeName = areaEntity.officeName,
        roomNumber = areaEntity.roomNumber
    )
}
